src/main/java/sirius/kernel/xml/XMLCall.java

```diff
@@ -26,7 +26,7 @@
  */
 public class XMLCall {
 
-    private Outcall outcall;
+    protected Outcall outcall;
```

Comment:
should not be needed...

Reply:
We can't define the URI otherwise as the outcall requires a URI at construction and this is only done in a static method that creates and returns a simple XMLCall. 

This is an issue in the design of the class, as the default constructor seems to be pointless as no endpoint can be defined that way. We can create a protected constructor that takes a URI and content type as parameters. This will be technically breaking but the default constructor effectively doesn't work outside the class anyway.